tests/integrations/virtual_environment/test_VirtualEnvironmentContext__executable.py

```diff
@@ -0,0 +1,51 @@
+import os
+from unittest.mock import patch
+
+import pytest
+
+from briefcase.integrations.virtual_environment import VenvContext
+
+
+class TestExecutable:
+    """Test cases for VenvContext.executable property."""
+
+    @pytest.mark.parametrize(
+        "platform, expected_filename",
+        [
+            ("nt", "python.exe"),
+            ("posix", "python"),
+            ("darwin", "python"),
+            ("linux", "python"),
+        ],
+    )
+    def test_executable_platform_specific_naming(
```

Comment:
You don't need to test every platform on every platform. CI runs tests on every platform, and we can lean on that. In this case, this should be 2 tests; one for windows behaviour, and one for non-windows behavior, with `pytest.skipif(sys.platform=="win32")` on tests to avoid the tests that can't pass.